tools/SourceKit/lib/SwiftLang/SwiftLangSupport.cpp

```diff
@@ -756,10 +762,36 @@ bool SwiftLangSupport::printAccessorUSR(const AbstractStorageDecl *D,
 
 std::string SwiftLangSupport::resolvePathSymlinks(StringRef FilePath) {
   std::string InputPath = FilePath;
+#if !defined(_WIN32)
   char full_path[MAXPATHLEN];
   if (const char *path = realpath(InputPath.c_str(), full_path))
     return path;
+
+  return InputPath;
+#else
+  char full_path[MAX_PATH];
+
+  HANDLE fileHandle = CreateFileA(
+      InputPath.c_str() /*lpFileName*/, GENERIC_READ /*dwDesiredAccess*/,
+      0 /*dwShareMode*/, nullptr /*lpSecurityAttributes */,
+      OPEN_EXISTING /*dwCreationDisposition*/,
+      FILE_FLAG_BACKUP_SEMANTICS |
+          FILE_FLAG_OPEN_REPARSE_POINT /*dwFlagsAndAttributes*/,
+      nullptr /*hTemplateFile*/);
```

Comment:
We generally don't do naming on the parameters unless they are trailing booleans.